client/src/components/ContentNavigator/index.ts

```diff
@@ -32,6 +32,18 @@ const fileValidator = (value: string): string | null =>
   )
     ? null
     : Messages.FileValidationError;
+
+const flowFileValidator = (value: string): string | null => {
+  let res = null;
+  if (!/^([^/<>;\\{}?#]+)\.\w+$/.test(value)) {
+    res = Messages.FileValidationError;
+  }
```

Comment:
Instead of duplication, we should call `fileValidator(value)` here

Reply:
done in f5253ce014a5fec429456474725f717e0f2609a0